static/app/views/automations/components/automationListRow.tsx

```diff
@@ -105,38 +93,4 @@ const RowWrapper = styled('div')<{disabled?: boolean}>`
       visibility: visible;
     }
   }
-
-  .last-triggered,
-  .action,
-  .connected-monitors {
-    display: none;
-  }
-
-  @media (min-width: ${p => p.theme.breakpoints.xsmall}) {
```

Comment:
These styles were being duplicated, but we don't need to do that if we use `subgrid`